runtime/themes/dark_plus.toml

```diff
@@ -1,138 +1,107 @@
-
-"warning" = { fg = "gold2" }
-"error" = { fg = "red" }
-"info" = { fg = "light_blue" }
-"hint" = { fg = "light_gray3" }
-
-"diagnostic.error".underline = { color = "red", style = "curl" }
-"diagnostic".underline = { color = "gold", style = "curl" }
+# UI
+"ui.background"            = { fg = "Unfocused", bg = "Back" }
+"ui.bufferline"            = { fg = "Unfocused", bg = "Back" }
+"ui.bufferline.active"     = { fg = "Front", bg = "Back", modifiers = ["underlined"]}
+"ui.bufferline.background" = { bg = "Back" }
+"ui.cursor"                = { fg = "Cursor", modifiers = ["reversed"] }
+"ui.cursor.match"          = { bg = "CursorMatch", modifiers = ["underlined"] }
+"ui.cursor.primary"        = { bg = "Cursor", modifiers = ["reversed"] }
+"ui.cursorline.primary"    = { bg = "Cursorline" }
+"ui.help"                  = { fg = "Front", bg = "PopupBack" }
+"ui.linenr"                = "Unfocused"
+"ui.linenr.selected"       = "Front"
+"ui.menu"                  = { fg = "Front", bg = "PopupBack" }
+"ui.menu.selected"         = { bg = "Selection" }
+"ui.popup"                 = { fg = "Front", bg = "PopupBack" }
+"ui.selection"             = { bg = "Selection" }
+"ui.selection.primary"     = { bg = "Selection" }
+"ui.statusline"            = { fg = "Front", bg = "StatusLine" }
+"ui.statusline.inactive"   = { fg = "Front", bg = "StatusLineNC" }
+"ui.text"                  = "Front"
+"ui.text.focus"            = "Front"
+"ui.virtual.indent-guide"  = "Unfocused"
+"ui.virtual.inlay-hint"    = "Green"
+"ui.virtual.ruler"         = { bg = "Unfocused" }
+"ui.virtual.whitespace"    = "WhiteSpace"
+"ui.window"                = { bg = "PopupBack" }
 
 [palette]
-white = "#ffffff"
-orange = "#ce9178"
-gold = "#d7ba7d"
-gold2 = "#cca700"
-pale_green = "#b5cea8"
-dark_green = "#6A9955"
-dark_green2 = "#487e02"
-light_gray = "#d4d4d4"
-light_gray2 = "#c6c6c6"
-light_gray3 = "#eeeeee"
-dark_gray = "#858585"
-dark_gray2 = "#1e1e1e"
-dark_gray3 = "#282828"
-dark_gray4 = "#404040"
-dark_gray5 = "#8b949e"
-blue = "#007acc"
-blue2 = "#569CD6"
-blue3 = "#6796E6"
-blue4 = "#1b81a8"
-light_blue = "#75beff"
-dark_blue = "#264f78"
-dark_blue2 = "#094771"
-red = "#ff1212"
-orange_red = "#f14c4c"
-
-type = "#4EC9B0"
-special = "#C586C0"
-variable = "#9CDCFE"
-fn_declaration = "#DCDCAA"
-constant = "#4FC1FF"
-
-background = "#1e1e1e"
-text = "#d4d4d4"
-cursor = "#a6a6a6"
-widget = "#252526"
-borders = "#323232"
+Front = '#D4D4D4'
+Back = '#1E1E1E'
+Blue = '#569CD6'
+Unfocused = '#5A5A5A'
+AccentBlue = '#4FC1FE'
+MediumBlue = '#18A2FE'
+LightBlue = '#9CDCFE'
+BlueGreen = '#4EC9B0'
+Green = '#6A9955'
+LightGreen = '#B5CEA8'
+Red = '#F44747'
+Orange = '#CE9178'
+YellowOrange = '#D7BA7D'
+Yellow = '#DCDCAA'
+Pink = '#C586C0'
+DiffRed = '#6F1313'
+DiffGreen = '#4B5632'
+DiffLightBlue = '#87d7ff'
+DiffBlue = '#005f87'
+SearchCurrent = '#515c6a'
+Search = '#613315'
+PopupFront = '#BBBBBB'
+PopupBack  = '#2D2D30'
+Selection = '#264F78'
+Cursor = '#A6A6A6'
+WhiteSpace = '#4D4D4D'
```

Comment:
Should be `Whitespace` set to `3e3e3d`

Reply:
Actually, I prefer to keep the names in terms of colors. That was originally my intention with this theme, but I never found the time to finish that. Trust me, its less of a headache trying to find a semantic name for things.